eZ/Publish/Core/settings/search_engines/solr/sort_clause_visitors.yml

```diff
@@ -66,7 +66,7 @@ services:
     ezpublish.search.solr.content.sort_clause_visitor.map_location_distance:
         class: %ezpublish.search.solr.content.sort_clause_visitor.map_location_distance.class%
         arguments:
-            - @ezpublish.search.solr.content.field_map
+            - @ezpublish.search.field_name_resolver
```

Comment:
Ideally we should maybe have "common" in the name here to keep it in sync with where the code is, but it's fine like this for now.